src/Adapter/Order/OrderAmountUpdater.php

```diff
@@ -292,6 +292,13 @@ private function getProductFromCart(array $cartProducts, int $productId, int $pr
 
             return $productMatch && $combinationMatch ? $item : null;
         });
+
+        // This shouldn't happen, if it does something was not done before updating the Order (removing an OrderDetail maybe)
```

Comment:
Missing one word in the comment 🤔 I think

Reply:
No, but maybe it's unclear indeed
```suggestion
        // This shouldn't happen, if it this happens it means something was not correctly done before calling OrderAmountUpdater::update (removing an OrderDetail maybe) and the cart and order are out of synce
```